### DISCUSSION TITLE
Skin editor elements positioned differently ingame.

### DISCUSSION THREAD

**Discussion by jkthdrpkck** [ppy/osu#26958] [Ideas]
When using the skin editor, the editing "window" is 4:3 so you can't see the entire gameplay area, you can really only see the playfield. This means when you want to place an element to the left or right of the playfield you have to kinda guess where it's going to be, due to that space being taken up by the layer/components section and the component settings section. It makes the process of adding things like your Combo, HP, Accuracy, etc (elements you wouldn't want inside the playfield) really tedious, and requires more trial and error than should be necessary.

Solution would be the just make the editing "window" fill to the aspect ratio of your screen, or adjust the locations of the layer/components and component settings sections - such as making them drop down tabs.

**Comment by j4chck** [comment]
Also in song select layout editor: 
![image](https://github.com/ppy/osu/assets/158584551/29cf53be-ce40-417a-b6f5-f29e676fc591)
![image](https://github.com/ppy/osu/assets/158584551/d6d3c556-b1df-459a-9bce-aa3a65934170)

**Comment by mixalisnew** [comment]
i have the same issue i would really love a fix to this

**Comment by plaguemask** [comment]
Make sure you're choosing the right Anchor and Origin. They determine how the element moves when the window size changes. They're shown with the red line and dot extending out from the element when it's selected in the editor. They're set automatically based on where the element is in the window, but you can choose them manually thru the right click menu.

In the screenshot, the star rating displays' anchors are probably `TopCentre`, meaning they're tied to the top center of the window. You probably want them tied to the top left corner of the window, so try changing their anchors to `TopLeft`.
